src/apprt/gtk/App.zig

```diff
@@ -428,7 +428,7 @@ pub fn init(core_app: *CoreApp, opts: Options) !App {
         // our "activate" call above will open a window.
         .running = c.g_application_get_is_remote(gapp) == 0,
         .css_provider = css_provider,
-        .custom_css_providers = std.ArrayList(*c.GtkCssProvider).init(core_app.alloc),
+        .custom_css_providers = .{},
     };
```

Comment:
Not necessary if you add a default value above, but it's more of a nitpick on my part.